server/views/partials/offencesPage/courtCaseContent.njk

```diff
@@ -0,0 +1,98 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% macro courtCaseContent(courtCase) %}
+
+<div class="govuk-accordion__section ">
+    <div class="govuk-accordion__section-header">
+      <h2 class="govuk-accordion__section-heading">
+        <span class="govuk-accordion__section-button" id="accordion-with-summary-sections-heading-1">
+          {{courtCase.courtName}}
+        </span>
+      </h2>
+      <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-1">
+        {{courtCase.sentenceDate}}
+      </div>
+    </div>
+    <div id="accordion-with-summary-sections-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-with-summary-sections-heading-1">
+    <h3>Offences</h3>
+    <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+        {% for offence in courtCase.offences %}
+            <li>
+                {{offence}}
+            </li>
+        {% endfor %}
+    </ul>
+
+    {% if courtCase.sentenceTerms.length > 0 %}
+        <h3>Sentences</h3>
```

Comment:
Is this the same level of information as the above `<h3>Offences</h3>` or should it be `h4`? (And then `h5` for the following subsections)
Same for the `else` part.